src/microbe_stage/editor/MicrobePartSelection.cs

```diff
@@ -156,7 +156,7 @@ private void UpdateLabels()
         if (mpLabel == null || nameLabel == null)
             return;
 
-        mpLabel.Text = MPCost.ToString(CultureInfo.CurrentCulture);
+        mpLabel.Text = MathUtils.RoundWithMinimalDecimals(MPCost, 5).ToString(CultureInfo.CurrentCulture);
```

Comment:
This 5 here should *probably* be a constant as this is likely needed in multiple places at some point.